### ISSUE TITLE
gpasswd -R

### ISSUE THREAD

**Issue by eagleoflqj** [shadow-maint/shadow#264]
The man page says with this option, only group members **with a password** can use newgrp to join the group. However, as per my test and [this link](https://bbs.archlinux.org/viewtopic.php?id=50739), **no password** is needed. Also, it indeed sets the password field at /etc/gpasswd to !, so **no password** can be used in theory.
Could you modify the man page to make it accurate?

**Comment by hallyn** [comment]
Yeah, I'll change that.  TBH I'm not sure what -R is meant to buy you (it doesn't even stop group admins from creating a new passwd), but clearly members can still newgrp to it without a password.
